generators/client-2/templates/src/main/webapp/app/account/settings/settings.html

```diff
@@ -1,83 +1,86 @@
-                           ng-model="vm.settingsAccount.email" ng-minlength=5 ng-maxlength=100 required maxlength="100">
-                    <div ng-show="form.email.$dirty && form.email.$invalid">
+                           [(ngModel)]="settingsAccount.email" minlength=5 maxlength=100 #emailInput="ngModel" required>
+                    <div *ngIf="emailInput.dirty && emailInput.invalid">
                         <p class="help-block"
-                               ng-show="form.email.$error.required" translate="global.messages.validate.email.required">
+                           *ngIf="emailInput.errors.required" translate="global.messages.validate.email.required">
                             Your e-mail is required.
                         </p>
                         <p class="help-block"
-                               ng-show="form.email.$error.email" translate="global.messages.validate.email.invalid">
+                           *ngIf="emailInput.errors.email" translate="global.messages.validate.email.invalid">
                             Your e-mail is invalid.
                         </p>
                         <p class="help-block"
-                               ng-show="form.email.$error.minlength" translate="global.messages.validate.email.minlength">
+                           *ngIf="emailInput.errors.minlength" translate="global.messages.validate.email.minlength">
                             Your e-mail is required to be at least 5 characters.
                         </p>
                         <p class="help-block"
-                               ng-show="form.email.$error.maxlength" translate="global.messages.validate.email.maxlength">
+                           *ngIf="emailInput.errors.maxlength" translate="global.messages.validate.email.maxlength">
                             Your e-mail cannot be longer than 100 characters.
                         </p>
                     </div>
-                </div><% if (enableTranslation){ %>
-                <div class="form-group">
+                </div>
+                <%_ if (enableTranslation){ _%>
+                <div class="form-group" *ngIf="languages && languages.length > 0">
                     <label for="langKey" translate="settings.form.language">Language</label>
-                    <select id="langKey" name="langKey" class="form-control" ng-model="vm.settingsAccount.langKey" ng-controller="<%=jhiPrefixCapitalized%>LanguageController as languageVm" ng-options="code as (code | findLanguageFromKey) for code in languageVm.languages"></select>
-                </div><% } %>
-
-                <button type="submit" ng-disabled="form.$invalid" class="btn btn-primary" translate="settings.form.button">Save</button>
+                    <select class="form-control" id="langKey" name="langKey" [(ngModel)]="settingsAccount.langKey">
+                        <option *ngFor="let language of languages" [value]="language">{{language}}</option>
```

Comment:
The `findLanguageFromKey `pipe needs to be moved from the `app.ng2module` to `shared.ng2module` and exported from there so that it can be used system wide